src/ui/components/TransferGrain.js

```diff
@@ -18,19 +20,74 @@ export type Props = {|
   +setLedger: (Ledger) => void,
 |};
 
+const useStyles = makeStyles({
+  root: {
+    width: "1100px",
+    margin: "0 auto",
+    background: "white",
+    padding: "0 5em 5em",
+  },
+  arrowBody: {
+    width: "300px",
+    marginLeft: "20px",
+    background: "#C9CED2",
+    padding: "5px 20px",
+  },
+  triangle: {
+    width: 0,
+    height: 0,
+    background: "white",
+    borderTop: "30px solid transparent",
+    borderBottom: "30px solid transparent",
+    borderLeft: "30px solid #C9CED2",
+  },
+  dropdown: {
+    width: "300px",
+    height: "50px",
+    border: "1px solid #9EADBA",
+    borderRadius: "500px",
+    background: "#F2F5F7",
+  },
+  centerRow: {
+    display: "flex",
+    justifyContent: "center",
+    alignItems: "center",
+  },
+  third: {
+    width: "300px",
+    height: "50px",
+    margin: "30px 15px",
+  },
+  half: {
+    width: "450px",
+    height: "50px",
+    margin: "30px",
+  },
+  memo: {
+    width: "960px",
+    height: "50px",
+    border: "1px solid #9EADBA",
+    paddingLeft: "20px",
+    color: "black",
+  },
+});
+
```

Comment:
Is this the bulk of all the styling work on this UI (I saw small bits of CSS along the way, too), or is there more elsewhere?

Reply:
This is the bulk, there are still a few one-off inline styles, but that's all. There is also the one theme declaration in the AccountSelector file. I am not very good with themes, but we will probably want to create custom theming for the material-ui components which we keep in some central file.